pyproject.toml

```diff
@@ -66,21 +68,13 @@ gdspy = {version="*", optional = true}
 gdstk = {version=">=0.9.49", optional = true}
 
 # jax
-jaxlib = [
-    {version = ">=0.4.1,<=0.4.14", platform = "linux",  source="jaxsource", optional = true},
-    {version = ">=0.4.1,<=0.4.14", platform = "darwin",  source="jaxsource", optional = true},
-    {version = ">=0.4.13,<=0.4.14", platform = "win32", source="jaxsource", python = "^3.9", optional = true}
-]
-jax = [
-    {version = ">=0.4.1,<=0.4.14", extras = ["cpu"], platform = "linux", source="jaxsource", optional = true},
-    {version = ">=0.4.1,<=0.4.14", extras = ["cpu"], platform = "darwin",  source="jaxsource", optional = true},
-    {version = ">=0.4.13,<=0.4.14", extras = ["cpu"], platform = "win32", python = "^3.9", source="jaxsource", optional = true}
-]
+jaxlib = {version = "^0.4.13,<=0.4.26", source="jaxsource", optional = true}
+jax = {version = "^0.4.13,<=0.4.26", extras = ["cpu"], source="jaxsource", optional = true}
 
 # trimesh
 networkx = {version = "^2.6.3", optional = true}
-rtree = {version = "1.0.1", optional = true}
-trimesh = {version = ">=4,<4.2", optional = true}
+rtree = {version = "1.2.0", optional = true}
+trimesh = {version = "^4", optional = true}
```

Comment:
Are you sure about this? Who set `trimesh < 4.2` before, @caseyflex ?

Reply:
I set it that way to match gdsfactory, which has since updated to `>=4, <4.4`. I think `^4` is better so we don't have to keep updating it. I don't know if the rtree version matters much as long as it is `>= 1.0.1`.

Reply:
Great!